wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1491,11 +1491,6 @@ data NodeSettings = NodeSettings {
    , setGitRevision    :: !Text
    } deriving (Show, Eq, Generic)
 
--- See note [Version Orphan]
-instance ToSchema Version where
-    declareNamedSchema _ =
-        pure $ NamedSchema (Just "Version") $ mempty
-            & type_ .~ SwaggerString
```

Comment:
We can delete [the note, too](https://github.com/input-output-hk/cardano-sl/blob/d8df53edd0ad81bfeb08ddad2c1dbeab1e2ba7d8/wallet-new/src/Cardano/Wallet/API/V1/Types.hs#L1495-L1498) :)

Reply:
Good catch!